OpenLaTeXStudio/EditorModule/src/latexstudio/editor/remote/DisconnectDropbox.java

```diff
@@ -0,0 +1,73 @@
+/* 
+ * Copyright (c) 2016 Sebastian Brudzinski
+ * 
+ * See the file LICENSE for copying permission.
+ */
+package latexstudio.editor.remote;
+
+import com.dropbox.core.DbxClient;
+import com.dropbox.core.DbxException;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.JOptionPane;
+import latexstudio.editor.ApplicationLogger;
+import latexstudio.editor.DropboxRevisionsTopComponent;
+import latexstudio.editor.TopComponentFactory;
+import latexstudio.editor.settings.ApplicationSettings;
+import latexstudio.editor.settings.SettingsService;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle.Messages;
+
+@ActionID(
+        category = "Remote",
+        id = "latexstudio.editor.remote.DisconnectDropbox"
+)
+@ActionRegistration(
+        displayName = "#CTL_DisconnectDropbox"
+)
+@ActionReference(path = "Menu/Remote", position = 3508)
```

Comment:
I didn't noticed the separator. I think ubuntu version doesn't show it (see the picture). However, I can see the difference in the code, so, which do you prefere?
I started putting it bellow the Connect item, and than I thought that seems more logical it to be at the end. So, if I put the separator below "Upload to Dropbox" it will work, right?